test/fixtures/rules/std_rule_cases/L009.yml

```diff
@@ -10,3 +10,7 @@ test_fail_no_final_newline:
 test_fail_multiple_final_newlines:
   fail_str: "SELECT foo FROM bar\n\n"
   fix_str: "SELECT foo FROM bar\n"
+
+test_pass_templated_plus_raw_newlines:
+  pass_str: |
+    {{ "\n\n" }}
```

Comment:
Why is this a valid pass_str? Should it be `{{ "\n\n" }}\n`?

Reply:
The vertical bar (`|`) character tells YAML this is a multiline string, and that we want to preserve newlines at the end of lines. Most other rules use multiline format, but this one mostly uses quoted strings. I'll change the test to the equivalent quoted string, as in your example above. 🙏🏽